public/js/utils.jsx

```diff
@@ -158,3 +158,12 @@ export function validateEmailAsYouType(email) {
     };
   }
 }
+
+export function hasErrorSubString(errorList, searchSubString) {
+  for (var i = 0; i < errorList.length; i += 1) {
```

Comment:
Hmm, lint is set to not like i++; We should probably change that but that's why.